pkg/controller/daemon/controller.go

```diff
@@ -316,8 +339,19 @@ func (dsc *DaemonSetsController) updateNode(old, cur interface{}) {
 		// A periodic relist will send update events for all known pods.
 		return
 	}
+	dsList, err := dsc.dsStore.List()
+	if err != nil {
+		glog.Errorf("Error enqueueing daemon sets: %v", err)
```

Comment:
The caller is an Informer. This signature of this func fits the event handler interface. There is no one below us that can handle this error. We need to handle it here.